Homepage.py

```diff
@@ -22,6 +22,8 @@
 
 today = date.today()
 default_date_yesterday = today - timedelta(days=1)

```

Comment:
Are those two lines still necessaries?

Reply:
Actually, I didn't catch that but I'll delete it.